cetmix_tower_yaml/static/description/index.html

```diff
@@ -435,32 +460,72 @@ <h2><a class="toc-backref" href="#toc-entry-7">17.0.1.0.1 (2025-05-05)</a></h2>
 </ul>
 </div>
 <div class="section" id="section-5">
+<<<<<<< HEAD
 <h2><a class="toc-backref" href="#toc-entry-8">17.0.1.0.0</a></h2>
 <p>Release for Odoo 17.0</p>
 </div>
 </div>
 <div class="section" id="bug-tracker">
 <h1><a class="toc-backref" href="#toc-entry-9">Bug Tracker</a></h1>
+=======
```

Comment:
This is left from merge conflict resolve, you must resolve the conflicts properly